packages/cardano-metamask-snap/src/fixtures/transactions.ts

```diff
@@ -77,4 +77,11 @@ export const transactionsFixture = {
       'f3caa64d97092841adc94a8b2c1a3d66a60c5358db08efb0c3f186efb9612f6a',
     networkId: 1,
   },
+  chang: {
```

Comment:
I'd consider making it clearer what this testcase is focused on, and I'd use the whole era name to be in line with Cardano specs/CDDL
```suggestion
  // contains nonempty set with tag 258 (https://github.com/IntersectMBO/cardano-ledger/blob/df783ed15f70b5b5c7a40dd5d8d62386f20bfec5/eras/conway/impl/cddl-files/conway.cddl#L220)
  conwayTaggedSet: {
```